script.js

```diff
@@ -3,24 +3,143 @@ const hamburgerImg = document.querySelector('#hamburger-img');
 const navMenu = document.querySelector('#hamburger-menu');
 const menuOptions = navMenu.querySelectorAll('a');
 const body = document.querySelector('body');
-const form = document.querySelector('.contact-form');
-const emailInput = document.querySelector('#email');
-const errorMsg = document.querySelector('#error-message');
-
-form.addEventListener('submit', (e) => {
-  e.preventDefault();
-  if (emailInput.value !== emailInput.value.toLowerCase()) {
-    errorMsg.textContent = 'Please enter your email in all lowercase!';
-  } else {
-    errorMsg.textContent = '';
-    form.submit();
-  }
+const portfolioGrid = document.querySelector('#portfolio-grid');
+const popup = document.querySelector('#popup');
+const workBtn = document.querySelector('.work-btn');
+const saveInfo = document.querySelector('.save-info');
+
+if (localStorage.getItem('myFormList') !== null) {
+  const dataList = JSON.parse(localStorage.getItem('myFormList'));
+  const lastForm = dataList[dataList.length - 1];
+
+  document.querySelector('#fname').value = lastForm.name;
+  document.querySelector('#email').value = lastForm.email;
+  document.querySelector('#subject').value = lastForm.message;
+}
```

Comment:
Well done for removing the errors in the previous PR.  I can see that the information is being stored in the localStorage, however it is still not being auto populated with the previous values in the input field. Kindly check and fix this. :wink:

Reply:
Hi @jerryowusu, in this project, this is what we were required to do:
- When the user loads the page, if there is any data in the local storage the input fields are pre-filled with this data.
When we reload the page, the input is pre-filled with the stored data from the local storage.